README.md

```diff
@@ -7,19 +7,22 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client resembling the classic F-Droid client.
```

Comment:
I'm not quite sure it's correct to replace it so. I'm not a native English speaker, but is sounds weird to me.

Reply:
Transitive verbs are just a neater way to relate things to each-other.
https://www.dictionary.com/browse/resembling

To illustrate a sentence that is a logic extreme of tediousness:

"The client of F-Droid that is unofficial is the one that resembles the client of F-Droid that is classic."

Reply:
> Transitive verbs are just a neater way to relate things to each-other.

Probably, but that's because "resembling" is not a common word. It can be replaced to something more common, I don't feel like "resembling" should be left here (though, I don't mind "that resembles", it feels normal to me).

Reply:
"That resembles" is not optimal, and not a common word. Will revise.